build.sbt

```diff
@@ -41,6 +41,7 @@ def projectWithPlayVersion(playVersion: PlayVersion) =
       "org.typelevel" %% "cats-core" % "2.9.0",
       commonsCodec,
       "org.scalatest" %% "scalatest" % "3.2.16" % Test,
+      "software.amazon.awssdk" % "ssm" % "2.21.7" % Test
```

Comment:
This new dependency is only for the test scope, as it's needed for the new `TwoFactorAuthCheckerCLITester`, only used for double-checking Google Credentials are correctly setup.